lms/djangoapps/ccx/views.py

```diff
@@ -559,8 +563,20 @@ def ccx_gradebook(request, course, ccx=None):
             for student in enrolled_students
         ]
 
+        paginator = Paginator(student_info, settings.MAX_STUDENTS_PER_PAGE)
```

Comment:
nit: is `MAX_STUDENTS_PER_PAGE` used only in the gradebook? if so, maybe we should use a specific name, otherwise leave it as is.